### PR TITLE
Dams ordered by latitude

### PR DESCRIPTION
**Pull request by elbfox**
Reordered decision alternatives, updated river rec labels, reordered dams by latitude, and revised interpretation text.

### CODE REVIEW

File: src/dams_mcda/WSM_graphs_test.R [hunk 1]
```diff
@@ -443,7 +444,7 @@ colnames(Dam2ScoreTable) <- criteria_inputs
 
 # Millinocket/Dam 3 output table(s)
 
-Dam3RawTable <- setDT(Mill_DataMatrix)
+Dam3RawTable <- setDT(East/Mill_DataMatrix)
```
Comment (sythel) [inline src/dams_mcda/WSM_graphs_test.R]: this might not be right

File: src/dams_mcda/ui.R [hunk 2]
```diff
@@ -1247,49 +1243,43 @@ ui <- shinyUI(fluidPage(
...
 					       #raw data table/Matrix
-					       h3("Table 14 Raw data values for East Millinocket Dam"),
+					       h3("Table 14 Raw data values for Dolby Dam"),
 					       DT::dataTableOutput("Dam4RawTable"),
 					       HTML(
 					         "<br><b>Results Interpretation</b> for Table 14. These are the raw data values for the dam development. We include the raw data values here to help make the MCDA calculation more transparent."
 					       ),
 					       # normalized data table/Matrix
-					       h3("Table 15. Normalized data values for East Millinocket Dam"),
+					       h3("Table 15. Normalized data values for Dolb Dam"),
```
Comment (sythel) [inline src/dams_mcda/ui.R]: Dolby

File: src/dams_mcda/WSM_graphs_test.R (hunk 1 above)
Comment (elbfox) [reply src/dams_mcda/WSM_graphs_test.R]: Thanks, I'm hoping to get to this momentarily.

File: src/dams_mcda/ui.R (hunk 2 above)
Comment (elbfox) [reply src/dams_mcda/ui.R]: Thanks, I'm hoping to get to this momentarily.

**Comment by elbfox** [comment]
corrected in forthcoming commits, just testing again locally

**Pull request merged**